R/new_matrixfit.R

```diff
@@ -238,6 +237,31 @@ NParticleModel <- R6Class(
   )
 )
 
+#' perform a factorising fit of a matrix of correlation functions
```

Comment:
@martin-ueding would be great if you could port any additional documentation that you might have from #169

Reply:
I don't think that there is any new documentation in that branch, I just looked through that.